Makefile.common

```diff
@@ -214,7 +221,7 @@ endif
 # Define the Subversion revision if available, either autodetected or
 # specified by the user, but only for base/version.cpp.
 ifneq ($(origin VER_REV), undefined)
-base/version.o: CXXFLAGS:=$(CXXFLAGS) -DSCUMMVM_REVISION=\"$(VER_REV)\"
+VERFLAGS = -DSCUMMVM_REVISION=\"$(VER_REV)\"
```

Comment:
I think this might need some code to do this as append to the variable, rather than overwrite assign ... as per the original code. Since otherwise, the define for AMIGAOS is likely to get lost...